Se habilito que el Usuario puedad remover imagenes de la galeria

felparodi opened the pull request:
![captura de pantalla de 2016-09-23 18 39 30](https://cloud.githubusercontent.com/assets/21043023/18802069/18991e0e-81bd-11e6-8532-fe8f530b5019.png)
![captura de pantalla de 2016-09-23 18 39 24](https://cloud.githubusercontent.com/assets/21043023/18802073/1aad68ee-81bd-11e6-904e-4a1998f0803a.png)

bernabe9 commented on src/containers/AnimalPerfilPage.js:
```diff
@@ -47,11 +70,19 @@ class AnimalPerfilPage extends Component {
               <p className="center">Galería</p>
               <AddGalleryButton animalId={this.props.routeParams.id}/>
+              { showButton &&
+              <button className={editGallery ? 'button-edit-galery active' : 'button-edit-galery'} onClick={this.editGallery}>
```
podes bajar `onClick` a la siguiente linea para que no quede tan larga

bernabe9 commented:
Mejoras visuales:
- Que se identifique mejor cuando el boton de editar esta apretado
- Mover el boton de editar junto con el de agregar, para poder aprovechar al maximo el ancho para mostrar la galeria
- Deshabilitar boton de agregar cuando se esta editando

bernabe9 commented:
BUG:
- Si elimino una foto, luego agrego una nueva no se refresca automaticamente

felparodi committed:
Se habilito que el Usuario puedad remover imagenes de la galeria

Conflicts:
	src/components/animals/ImagesGallery.js
	src/containers/AnimalPerfilPage.js
